buildSrc/src/main/groovy/bio.terra.catalog.java-common-conventions.gradle

```diff
@@ -41,15 +41,15 @@ repositories {
 
 dependencies {
     compileOnly 'com.github.spotbugs:spotbugs-annotations:4.7.2'
-    implementation 'io.swagger.core.v3:swagger-annotations:2.2.0'
-    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.31'
-
+    implementation 'io.swagger.core.v3:swagger-annotations:2.2.12'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.47'
```

Comment:
With this update I think the `jvmArgs = ['--add-opens=java.base/java.util=ALL-UNNAMED']` fix can be removed (it's in three places), you could try that now or we can do it later on.